models/base/ga/ga_stats.sql

```diff
@@ -64,10 +64,6 @@ sum(goal_completions_all_goals_ttm) as goal_completions_all_goals_ttm,
 max(goal_conversion_rate_all_goals_ttm) as goal_conversion_rate_all_goals_ttm,
 max(bounce_rate_ttm) bounce_rate_ttm,
 max(avg_seconds_on_site_ttm) as avg_seconds_on_site_ttm,
-CASE WHEN sum(transactions_30d) > 0 THEN sum(transactions_30d)
```

Comment:
Didn't we decide on leaving blended conversions in the models (for use in action recommendations on a line-level), but also recalculating them as calculated fields in DS for aggregated reporting purposes?